community.md

```diff
@@ -16,19 +40,9 @@ R-multiverse always provides a version that its developer chose to distribute fo
 2. The user is still responsible for judging whether a package is appropriate to use.
 Some releases may still have issues, as indicated by their check results, publicly available at <https://community.r-multiverse.org>.
 
-## Continuous maintainer-driven deployment
-
-Anyone can [contribute](contributors.md) packages to Community via GitHub.
-Each package only needs to be registered once.
-
-Every new [release](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) then automatically deploys to the repository without needing to go through R-multiverse again.
-In other words, the package maintainer is in complete control.
-
-This frictionless, maintainer-driven experience was made possible by the incredible infrastructure of [rOpenSci's R-universe system](https://ropensci.org/r-universe/).
-
-## What about my personal R-universe?
-
-R-multiverse perfectly complements your existing personal or organization R-universe.
+## Personal universes
 
-This can continue to hold either development or release versions for yourself and your colleagues.
-At the same time, R-multiverse deploys the latest package releases directly to the wider community.
+You can deploy a package through a personal or [organization](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations)-level
+universe even if the package also is part of R-multiverse.
+In fact, a personal universe may help reproduce and diagnose `R CMD check` issues
```

Comment:
later maybe show an actual example of a package with such a setup / or add a diagram?

Reply:
8b085bc4612020e2480092595e9b5de7633619d0 adds a link to where the Production page elaborates on this.